DigitalLearningSolutions.Web/Styles/trackingSystem/delegates/delegateCommon.scss

```diff
@@ -1,7 +1,7 @@
 ﻿@import "~nhsuk-frontend/packages/core/all";
 @import "../../shared/breakpoints";
 
-.all-delegates-button-group {
+.delegate-heading-button-group {

```

Comment:
I feel like these buttons are generic enough to not only be associated with delegate pages